components/Comments.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../src/api";
+import CommentCard from "./CommentCard";
+
+export default function Comments({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((comment) => {
```

Comment:
are there any errors from getting the comments for an article? Try viewing `/articles/37` and see what happens!